samples/subsys/mgmt/mcumgr/smp_svr/boards/nrf52840dk_nrf52840 copy.conf

```diff
@@ -0,0 +1,15 @@
+# Allow for large Bluetooth data packets.
```

Comment:
This file seems to be weird named copy of samples/subsys/mgmt/mcumgr/smp_svr/boards/nrf52840dk_nrf52840.conf,  is that ok?

Reply:
thx - this intended to be nrf52dk one.